client/my-sites/site-settings/settings-security/main.jsx

```diff
@@ -63,20 +64,34 @@ const SiteSettingsSecurity = ( { rewindActive, site, siteId, siteIsJetpack, tran
 
 	return (
 		<Main className="settings-security__main site-settings">
-			<QueryRewindStatus siteId={ siteId } />
+			<QueryRewindState siteId={ siteId } />
 			<DocumentHead title={ translate( 'Site Settings' ) } />
 			<JetpackDevModeNotice />
 			<SidebarNavigation />
 			<SiteSettingsNavigation site={ site } section="security" />
-			{ rewindActive && <Backups /> }
+			{ ( includes( [ 'active', 'awaitingCredentials', 'provisioning' ], rewindState.state ) ||
+				( 'unavailable' === rewindState.state &&
+					'vp_active_on_site' === rewindState.failureReason ) ) && (
+				<Backups
```

Comment:
This might be a good time to rename this. I'm not sure that `Backups` is clear to someone who hasn't already worked on this - maybe something like `RewindCredentials` or `JetpackCredentials`?